internal/tiltfile/docker.go

```diff
@@ -365,6 +334,25 @@ func (b *customBuild) addFastBuild(thread *starlark.Thread, fn *starlark.Builtin
 	return &fastBuild{s: b.s, img: b.img}, nil
 }
 
+func parseValuesToStrings(value starlark.Value, s string) ([]string, error) {
```

Comment:
Maybe rename this function to `valueToStringsForParam` so it's more obvious what `s` is for?

```go
func valueToStringsForParam(value starlark.Value, param string)
```